src/components/Auction.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Timer from "./Timer.jsx";
+import { RiCoinLine } from "react-icons/ri";
+import { currentAutcions } from "../mocks/auctions.js";
+
+const Auction = ({ id, name, expiration, highest_bid, image }) => {
+    return (
+        <>
+            <div className="container__small">
```

Comment:
Deze container moet eigenlijk rond de map-functie in de Index.jsx

Reply:
^